lib/ansible/modules/system/parted.py

```diff
@@ -416,7 +417,7 @@ def get_device_info(device, unit):
         return get_unlabeled_device_info(device, unit)
 
     command = "%s -s -m %s -- unit '%s' print" % (parted_exec, device, unit)
-    rc, out, err = module.run_command(command)
+    rc, out, err = module.run_command(command, environ_update=env_default)
```

Comment:
instead of calling it for every command, you can just use `module.run_command_environ_update` to set it once